tools/b43-tools/Makefile

```diff
@@ -8,18 +8,17 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=b43-tools
-PKG_DATE:=2017-09-13
+PKG_DATE:=2022-07-05
 
 PKG_SOURCE_URL:=https://github.com/mbuesch/b43-tools.git
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)
-PKG_SOURCE_VERSION:=27892ef741e7f1d08cb939744f8b8f5dac7b04ae
+PKG_SOURCE_VERSION:=2fe10ea6690df9a068cb21cde537236bae784a14
 PKG_MIRROR_HASH:=f914c36ac566e9e3b5a3a04de16ddb014fcad6a1cf25cdd8e4825c708d28d3f4
```

Comment:
The PKG_MIRROR_HASH is wrong and needs an update.

Reply:
Oh, we need to remove PKG_SOURCE_SUBDIR to use git download instead of sources.cdn.openwrt.org